dynamic-layers/variscite-nxp-layer/recipes-bsp/u-boot/u-boot-variscite/0001-Add-hmx1-device-tree-for-u-boot.patch

```diff
@@ -1,35 +1,60 @@
-From fe13b93afce8d96b37a3a4618384d054f1e354cb Mon Sep 17 00:00:00 2001
-From: rikardo <rikard.olander@hostmobility.com>
-Date: Wed, 22 Feb 2023 07:28:59 +0000
-Subject: [PATCH 1/4] Add-hmx1-device-tree-for-u-boot
+From fd4923ac3c04dfcb0252cee6baea46a777d93b13 Mon Sep 17 00:00:00 2001
+From: Andreas Ternstedt <a.ternstedt@setek.se>
+Date: Thu, 14 Aug 2025 17:23:43 +0200
+Subject: [PATCH 1/4] arm: dts: Add HMX1 device tree for U-Boot
 
-v1.
-Change serial console
- port to uart3 Disable fec
+Add device tree for HMX1 board based on imx8mp-var-dart with
+HMX-specific hardware configuration:
+- Change serial console from UART1 to UART3
+- Disable EQOS & FEC ethernet
+- Enable USB hubs on boot with proper GPIO configuration
+- Add HMX EEPROM support
+- Fix SD card detection with correct pinctrl settings
+
+Based on Variscite U-boot version: lf_v2024.04_6.6.52-2.2.0_var01
+
+Co-developed-by: rikardo <rikard.olander@hostmobility.com>
+Signed-off-by: Andreas Ternstedt <a.ternstedt@setek.se>
+
+---
+
+Notes:
+	v6: Remove EQOS & FEC, no support of marvell 88q2xxx in Uboot
+    v5: Adapted patch for Variscite U-boot lf_v2024.04_6.6.52-2.2.0_var01
+    v4: - Changed co-cpu binary name
+        - Added DDR memory load for co-cpu
+        - Delayed M7 firmware loading until after flashing
+    v3: - Added product-number to device tree
+        - Added RGB LED controller register to device tree
+        - Added M7 co-processor support (hmx-m7-fw.bin)
+    v2: - Added HMX EEPROM environment variables
+        - Added serial/part numbers to device tree
+        - Added USB upgrade button support
+        - Changed BSP script to hmx_boot.scr
+        - Added RGB LED handling
+        - Added watchdog initialization
+    v1: - Changed UART1 to UART3
+        - Updated U-Boot environment for HMX device tree
 
- v2.
- Fix so usb hubs is actived on boot. Add eeprom
- support.
- Fix so sdcard can be detected (change pinctrl)
 ---
- .../dts/imx8mp-var-dart-dt8mcustomboard.dts   | 103 +++++++++++-------
- 1 file changed, 62 insertions(+), 41 deletions(-)
+ .../dts/imx8mp-var-dart-dt8mcustomboard.dts   | 102 +++++++++---------
+ 1 file changed, 53 insertions(+), 49 deletions(-)
 
 diff --git a/arch/arm/dts/imx8mp-var-dart-dt8mcustomboard.dts b/arch/arm/dts/imx8mp-var-dart-dt8mcustomboard.dts
-index 1d683d9344..9ee25a53df 100644
+index edb25d64cbb..5883414adb9 100644
 --- a/arch/arm/dts/imx8mp-var-dart-dt8mcustomboard.dts
 +++ b/arch/arm/dts/imx8mp-var-dart-dt8mcustomboard.dts
 @@ -2,20 +2,16 @@
  /*
   * Copyright 2019 NXP
-  * Copyright 2020-2022 Variscite Ltd.
-+ * Copyright 2022 Host Mobility AB.
```

Comment:
You cannot have Copyright 2025 Host Mobility AB because that company no longer exists. Instead, add a new Copyright 2025 SETEK Systems AB
Also, check history to see if any changes were made in 2023 and 2024